test/blocks/marketo-config/marketo-config.test.js

```diff
@@ -141,4 +139,22 @@ describe('marketo-config', () => {
     const copyContent = copyBtn.querySelector('.copy-content');
     expect(copyContent.textContent).to.contain('http');
   });
+
+  it('resets to default state', async () => {
+    const el = document.querySelector('.marketo-config');
+    await init(el);
+
+    const accordion = await waitForElement('.accordion');
+
+    const resetButton = accordion.querySelector('.resetToDefaultState');
+    resetButton.click();
+
+    await delay(50);
```

Comment:
Why is the delay needed? That seems like it will lead to a flaky test. Either wait for an element to appear, or run all the async code so you can be sure something executed

```
  const clock = sinon.useFakeTimers({
    // Intercept setTimeout and call the function immediately
    toFake: ['setTimeout'],
  });
  await clock.runAllAsync();
  clock.restore();
````